src/pfs/pfs.proto

```diff
@@ -284,6 +294,31 @@ message DeleteBranchRequest {
   bool force = 2;
 }
 
+message CreateProjectRequest {
+  Project project = 1;
```

Comment:
Should this be a Project or a ProjectInfo?

Reply:
to be consistent with other Request types this should be Project